.clang-format

```diff
@@ -260,6 +260,31 @@ StatementMacros:
   - _END_EXTERN_C
   - _EXTERN_C_UNLESS_PURE
   - _END_EXTERN_C_UNLESS_PURE
+  - _NODISCARD_CTOR_LOCK
+  - _NODISCARD_TRY_CHANGE_STATE
+  # SAL
+  - _When_
+  # Concurrency SAL
+  - _Requires_lock_held_
+  - _Requires_exclusive_lock_held_
+  - _Requires_shared_lock_held_
+  - _Requires_lock_not_held_
+  - _Requires_no_locks
+  - _Acquires_lock_

```

Comment:
`_Acquires_lock_` is unused. We're using either `_Acquires_exclusive_lock_` or `_Acquires_shared_lock_`, so I think we should drop the plain one.

Reply:
It will be used for std::mutex.
I can remove it in this PR and add in another one.